pkg/apiserver/registry/inventory/vpc/rest.go

```diff
@@ -38,6 +38,20 @@ import (
 	"antrea.io/nephe/pkg/labels"
 )
 
+const (
+	cloudAccountName      = labels.CloudAccountName
+	cloudAccountNamespace = labels.CloudAccountNamespace
+	metaName              = "metadata.name"
+	metaNamespace         = "metadata.namespace"
+	statusCloudId         = "status.cloudId"
+	statusRegion          = "status.region"
+)
+
+const (
+	numFieldSelectors = 4
```

Comment:
same here.. replace the const with enum and do size of to get len

Reply:
Done